### PR TITLE
update placement with extensible scheduling

### PR DESCRIPTION
**Pull request by haoqing0110**
The related feature request: open-cluster-management-io/community#52
Ref: https://github.com/open-cluster-management/backlog/issues/18210

Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

**Comment by haoqing0110** [comment]
@qiujian16 @elgnay plz help review this doc.

File: content/en/concepts/placement.md [hunk 1]
```diff
@@ -74,21 +74,40 @@ spec:
...
-  - `name` is the name of a prioritizer. Below are the valid names:
-    - Balance: balance the decisions among the clusters.
-    - Steady: ensure the existing decision is stabilized.
-    - ResourceAllocatableCPU & ResourceAllocatableMemory: sort clusters based on the allocatable.
-  - `weight` defines the weight of prioritizer. The value must be ranged in [0,10].
+  - `scoreCoordinate` represents the configuration of the prioritizer and score source.
+    - `type` defines the type of the prioritizer score.
+    Type is either "BuiltIn", "AddOn" or "", where "" is "BuiltIn" by default.
+	  When the type is "BuiltIn", need to specify a BuiltIn prioritizer name in BuiltIn.
+    When the type is "AddOn", need to configure the score source in AddOn.
```
Comment (qiujian16) [inline content/en/concepts/placement.md:97]: alignment seems incorrect.

File: content/en/concepts/placement.md [hunk 2]
```diff
@@ -74,21 +74,40 @@ spec:
...
 - `configurations` represents the configuration of prioritizers.
-  - `name` is the name of a prioritizer. Below are the valid names:
-    - Balance: balance the decisions among the clusters.
-    - Steady: ensure the existing decision is stabilized.
-    - ResourceAllocatableCPU & ResourceAllocatableMemory: sort clusters based on the allocatable.
-  - `weight` defines the weight of prioritizer. The value must be ranged in [0,10].
+  - `scoreCoordinate` represents the configuration of the prioritizer and score source.
+    - `type` defines the type of the prioritizer score.
+    Type is either "BuiltIn", "AddOn" or "", where "" is "BuiltIn" by default.
+	  When the type is "BuiltIn", need to specify a BuiltIn prioritizer name in BuiltIn.
```
Comment (qiujian16) [inline content/en/concepts/placement.md:96]: a BuiltIn prioritizer name must be specified

File: content/en/concepts/placement.md [hunk 3]
```diff
@@ -109,6 +128,15 @@ spec:
 
 `PlacementDecision` can be consumed by another operand to decide how the workload should be placed in multiple clusters.
 
+## Extensible scheduling
+In placement resource based scheduling, in some cases the prioritizer needs extra data (more than the default value provided by ManagedCluster) to calculate the score of the managed cluster. For example, there is a requirement to schedule based on resource monitoring data from the cluster.
```
Comment (qiujian16) [inline content/en/concepts/placement.md:132]: For example, schedule the clusters based on cpu or memory usage data of the clusters fetched from a monitoring system

File: content/en/concepts/placement.md (hunk 1 above)
Comment (haoqing0110) [reply content/en/concepts/placement.md:97]: fixed.

File: content/en/concepts/placement.md (hunk 2 above)
Comment (haoqing0110) [reply content/en/concepts/placement.md]: updated.

File: content/en/concepts/placement.md (hunk 3 above)
Comment (haoqing0110) [reply content/en/concepts/placement.md]: br_placement.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**